www/apps/book/app/page.mdx

```diff
@@ -44,13 +45,11 @@ npx create-medusa-app@rc
 
 ---
 
-## Who is Medusa for
+## Who should use Medusa
 
-Medusa is for ambitious businesses and developers that are limited by traditional ecommerce solutions:
+Medusa is for businesses that have unique requirements that other digital commerce platforms can't easily support. Businesses of any size can use Medusa, from small startups to large enterprises - all it takes is a developer who can run a few commands and deploy your project.
 
-- You want rich commerce features that are customizable and flexible enough to support your business use case as if you built them from scratch.
-- You want your custom requirements built as an integral part of the commerce application’s features and architecture, rather than hacked into it.
-- You want to customize your commerce flows by integrating external services without data inconsistency and irregularity.
+Below are some stories from companies that use Medusa:
 
 <CardList items={[
   {
```

Comment:
would really like these to use logos of the companies to make them feel more user story-ish.

Reply:
I can handle this in a separate PR 👍🏻